src/components/Pages/LoginPage/utils.ts

```diff
@@ -0,0 +1,92 @@
+import { ethers } from 'ethers'
+import { AuthIdentity, Authenticator } from '@dcl/crypto'
+import { ProviderType } from '@dcl/schemas/dist/dapps/provider-type'
+import { getIdentity, storeIdentity } from '@dcl/single-sign-on-client'
+import { connection, ConnectionResponse, Provider } from 'decentraland-connect'
+import { config } from '../../../modules/config'
+import { ConnectionOptionType } from '../../Connection'
+
+const ONE_MONTH_IN_MINUTES = 60 * 24 * 30
+
+const MAGIC_KEY = config.get('MAGIC_KEY')
+
+export function fromConnectionOptionToProviderType(connectionType: ConnectionOptionType) {
+  switch (connectionType) {
+    case ConnectionOptionType.DISCORD:
+    case ConnectionOptionType.X:
+    case ConnectionOptionType.GOOGLE:
+    case ConnectionOptionType.APPLE:
+      return ProviderType.MAGIC
+    case ConnectionOptionType.WALLET_CONNECT:
+    case ConnectionOptionType.METAMASK_MOBILE:
+      return ProviderType.WALLET_CONNECT_V2
+    case ConnectionOptionType.COINBASE:
```

Comment:
I think coinbase provider should be injected given that they also have a wallet extension for the browser.

Reply:
I'll keep it this way as this is copy of the logic that is built on the `decentraland-dapps` repository